report-export/json-export-d3/demo.html

```diff
@@ -1,11 +1,9 @@
-<script src="https://code.jquery.com/jquery-2.1.0.js"></script>
+
 <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.4.8/d3.js"></script>
 <script src="https://rawgit.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js"></script>
```

Comment:
[rawgit.com](rawgit.com) reached end of it's life.
probably it's better to replace with direct github link: [https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js](https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js)
and even better it to fork this repo to tibco github in order to not lost it.

Reply:
After replacing, fiddle shows "Cross-Origin Read Blocking (CORB) blocked cross-origin response https://raw.githubusercontent.com/zigfred/dimple/master/dist/dimple.v2.1.2tmp.js with MIME type text/plain. See https://www.chromestatus.com/feature/5629709824032768 for more details." this error in console.

Reply:
we can put that script to github pages